src/estimate/ml-metrorun.c

```diff
@@ -298,47 +353,51 @@ int main(int argc, char *argv[]) {
   printDataTypeIndices(dataTypeIndices, numDataTypes, userOut);
   fprintf(userOut, "\n\n");
 
-  printDataTypeWeightIndices(dataTypeIndices, numDataTypes, dataTypeWeights,userOut);
+  printDataTypeWeightIndices(dataTypeIndices, numDataTypes, dataTypeWeights,
+                             userOut);
   fprintf(userOut, "\n\n");
 
-  readData(inFileName, dataTypeIndices, numDataTypes, MAX_DATA_TYPES, numLocs, steps,
-	   validFrac, optIndicesFile, compareIndicesFile, userOut);
+  readData(inFileName, dataTypeIndices, numDataTypes, MAX_DATA_TYPES, numLocs,
+           steps, validFrac, optIndicesFile, compareIndicesFile, userOut);
 
-  if (strcmp(aggregationFile, "") != 0) { // there is a file for model-data aggregation
+  if (strcmp(aggregationFile, "") != 0) {  // there is a file for model-data
+                                           // aggregation
     readFileForAgg(aggregationFile, numDataTypes, unaggedWeight);
     differenceFunc = aggedDifference;
-    numDataTypes *= 2; /* A bifurcation of data types: each data type is split into two data types:
-			  an aggregated and an unaggregated.
-			  But this bifurcation only applies to the collection of statistics, not
-			  to the actual running of the model!
-			  We'll have to "remember" this in the aggedDifference function!
-		       */
+    /* A bifurcation of data types; each data type is split into two data types:
+     *  an aggregated and an unaggregated. But this bifurcation only applies to
+     *  the collection of statistics, not to the actual running of the model!
+     *  We'll have to "remember" this in the aggedDifference function! */
+    numDataTypes *= 2;
   }
 
-  signal(SIGINT,exit);
+  signal(SIGINT, exit);
   seedRand(0, userOut);
 
   for (runNum = 1; runNum <= numRuns; runNum++) {
     fprintf(userOut, "Run #%d of %d:\n\n", runNum, numRuns);
 
-    if (numRuns == 1)
-      strcpy(thisFile, outFileName); // just use base file name
-    else
-      sprintf(thisFile, "%s_%d_", outFileName, runNum); /* append runNum to base file name
-							   add extra underscore at end to separate run # from location #
-							*/
+    if (numRuns == 1) {
+      strcpy(thisFile, outFileName);  // just use base file name
+    } else {
+      sprintf(thisFile, "%s_%d_", outFileName, runNum);
+    } /* append runNum to base file name
+                      add extra underscore at end to separate run # from
+         location #
+                   */
```

Comment:
Yet another I'd fix if there were one of them but not for the whole codebase at once